### PR TITLE
Scintilla v5.5.0

### PR DESCRIPTION
**Pull request by ahmetsait**
Changes of interest:
- Implement solution-local NuGet registry
- Remove unused project files and put source files into project folder
- Move `build/**.dll` to NuGet's special `runtimes` folder
- Update `Scintilla.dll` & `Lexilla.dll`
- Fix transparent colors
- Fix incorrect default values
- Implement Scintilla element APIs

### CODE REVIEW

**Comment by ahmetsait** [comment]
Published as [ahmetsait.Scintilla5.NET](https://www.nuget.org/packages/ahmetsait.Scintilla5.NET/5.5.0) for easy testing.

**Comment by ahmetsait** [comment]
Rebased.
I know it's a lot to take at once but all changes are contained in their atomic commits for easy review so it would be phenomenal if this gets merged.

**Review by desjarlais — APPROVED** [review]

File: Scintilla.NET/Scintilla.NET.csproj [hunk 1]
```diff
@@ -1,179 +1,62 @@
...
-		<AssemblyTitle>ScintillaNET</AssemblyTitle>
-		<Company>Jacob Slusser, VPKSoft, cyber960, desjarlais</Company>
-		<Product>Scintilla.NET</Product>
+		<TargetFrameworks>net462;net6.0-windows</TargetFrameworks>
 		<NeutralLanguage>en-US</NeutralLanguage>
 		<Description>Source Editing Component based on Scintilla 5 series.</Description>
-		<Copyright>Copyright (c) 2018, Jacob Slusser. All rights reserved. VPKSoft, cyber960 2022, desjarlais 2023.</Copyright>
-		<Version>5.3.3.19</Version>
+		<Copyright>Copyright (c) Jacob Slusser 2018, VPKSoft, cyber960 2022, desjarlais 2023.</Copyright>
+		<Version>5.5.0</Version>
```
[the remark this answers is not in the document]
Comment (ahmetsait) [reply Scintilla.NET/Scintilla.NET.csproj:8]: Don't have a good reason other than *it seemed close enough and a nice round number*. Agreed that they are unrelated and we shouldn't be trying to sync them.

**Comment by ahmetsait** [comment]
Thanks a lot! Any chance of a NuGet release?

**Comment by desjarlais** [comment]
@ahmetsait I'm getting build errors with the nuget file with the cd command giving an exit code 1.  One I fix that I'll work on a release.

**Comment by desjarlais** [comment]
I got it fixed, ps was disabled so the ps1 was not able to run, it is good now the nuget is released

**Pull request merged**